pycds/alembic/versions/7b139906ac46_add_not_null_constraint_on_meta_vars.py

```diff
@@ -0,0 +1,85 @@
+"""add not null constraint on meta vars
+
+Revision ID: 7b139906ac46
+Revises: 3d50ec832e47
+Create Date: 2022-01-17 14:21:37.975335
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from pycds import get_schema_name
+
+# revision identifiers, used by Alembic.
+revision = "7b139906ac46"
+down_revision = "3d50ec832e47"
+branch_labels = None
+depends_on = None
+
+schema_name = get_schema_name()
+table_name = "meta_vars"
+
+
+def update_table(
```

Comment:
Good factoring. 

I think Python Black would reformat these method defs.

Reply:
I ran black on this file and it did not change anything.

Reply:
That's odd. I ran Black on it and it made quite a few changes, consistent with the [Black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html), which this file is not, unless you specified a very short line length. Try running `black --line-length 80 ...`.

Don't spend much time on this, however. Code formatting matters, but we don't want to get bogged down in it.

Reply:
I ran that ```black --line=length 80 pycds/alembic/versions/7b139906ac46_add_not_null_constraint_on_meta_vars.py``` and still got the 1 file left unchanged result.